portal-web/test/functional/com/liferay/portalweb/macros/headless/journal/HeadlessWebcontentFolderAPI.macro

```diff
@@ -147,10 +176,12 @@ definition {
 		return "${response}";
 	}
 
-	macro getIdOfCreateStructuredContentFolder {
-		var folderId = JSONUtil.getWithJSONPath("${responseToParse}", "$.id");
+	macro getElementOfCreatedStructuredContentFolder {
```

Comment:
Renamed to getElementOf..., so that we can get both id and parentStructuredContentFolderID in our tests.